include/alpaka/api/unifiedCudaHip/Queue.hpp

```diff
@@ -125,6 +125,11 @@ namespace alpaka::onHost
 
             friend struct onHost::internal::Enqueue;
 
+            void enqueueHostFnAsync(auto const& task)
+            {
+                m_callBackThread.submit(task);
```

Comment:
I assume this should be removed because you spezialized the trait `HostTaskAsync`